stratify/_bounded_vinterp.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+from ._conservative import conservative_interpolation
+
+
+def interpolate(z_target, z_src, fz_src, axis=-1):
+    """
```

Comment:
I've suggested renaming this in another comment.  If you opt to retain the existing name, I'd suggest adding that the interpolation is conservative to this docstring?